templates/cronjob.yaml

```diff
@@ -56,6 +56,10 @@ spec:
                 secretKeyRef:
                   name: {{ .Values.s3.secret.name }}
                   key: {{ .Values.s3.secret.keys.secretAccessKey }}
+{{- if .Values.s3.endpoint.enabled }}
```

Comment:
Could we just check whether the endpoint url value is null? Maybe `s3.endpointUrl` or `s3.endpoint`?

Reply:
sure. I just changed it.